Makefile

```diff
@@ -0,0 +1,82 @@
+.PHONY: db
+
+DATA_LINK="https://s3.amazonaws.com/instacart-datasets/instacart_online_grocery_shopping_2017_05_01.tar.gz"
+# Overwrite this by running `make BUCKET="my-test-bucket" s3` for example
+BUCKET="instacart-store"
+# MODE can be "local" or "rds"
+# Overwrite this by running `make MODE="rds" db` for example
+MODE="local"
+
+data/external/raw_data.tar.gz:
+	curl -X GET ${DATA_LINK} -o data/external/raw_data.tar.gz && tar -xvzf data/external/raw_data.tar.gz -C data/external && mv data/external/instacart_2017_05_01/* data/external && rm -rf data/external/instacart_2017_05_01
```

Comment:
Does pep8 apply for long makefile commands and URL? Just a note.

Reply:
Lol nice try but they are separate commands if on different lines

Reply:
wait actually I think you can maybe wrap them in `()` and use `\` for new lines... I'll try